cwr/convert.go

```diff
@@ -42,31 +57,206 @@ func NewConverter(store *meta.Store) *Converter {
 	}
 }
 
-// ConvertRegisteredWork reads all registeredWork , converts them to META
-// objects, stores them in the META store and sends their CIDs to the given
-// stream.
-func (c *Converter) ConvertRegisteredWork(ctx context.Context, outStream chan *cid.Cid, cwrFileReader io.Reader) error {
-	// get records from the db
-	records, err := ParseCWRFile(cwrFileReader)
+// Transaction struct
+type Transaction struct { // NWR,REV,EXC,ACK,AGR or ISW transaction
+	//Each transaction include one main record and few details records.
+	//a transaction can include few details records from the same type
```

Comment:
It is idiomatic for Go comments to begin with what is being described, for example `MainRecord is ...`, see [here](https://blog.golang.org/godoc-documenting-go-code).